docs/en/stack/ml/df-analytics/ml-dfa-phases.asciidoc

```diff
@@ -29,17 +29,30 @@ Once the destination index is built, the {dfanalytics-job} task calls the {es}
 ==== Loading data
 
 After the reindexing is finished, the job fetches the needed data from the 
-destination index. It converts the data into the format that the analysis process 
-expects, then sends it to the analysis process.
+destination index. It converts the data into the format that the analysis 
+process expects, then sends it to the analysis process.
 
 
 [discrete]
 ==== Analyzing
 
-When all the required data is loaded, the analyzing phase begins. The exact 
-process always depends on the type of {dfanalytics} – for example, 
-{classification} or {oldetection} – that is executed. This is the phase when the 
-job generates a {ml} model for analyzing the dataset.
+In this phase, the job generates a {ml} model for analyzing the data. The 
+specific phases of analysis vary depending on the type of {dfanalytics-job}.
+
+{oldetection-cap} jobs have a single analysis phase called `computing_outliers`, 
+in which they identify outliers in the data.
+
+{regression-cap} and {classification} jobs have four analysis phases:
+
+. `feature_selection`: Identifies which fields are significant for the analysis. 
+. `coarse_parameter_search`: The first part of 
+  <<hyperparameters,hyperparameter optimization>> in which the process finds 
+  suitable values for the hyperparameters that are required to run the analysis. 
+. `fine_tuning_parameters`: After the values of hyperparameters are narrowed 
+  down to a certain range in the `coarse_parameter_search` phase, the parameters 
+  are further refined to an exact value in this second part of hyperparameter 
+  optimization.
+. `final_training`: This is the phase where the {ml} model training takes place.
```

Comment:
If you want to make these more concise, you can make them partial phrases. For example:
```suggestion
. `final_training`: Trains the {ml} model.
```